doc/en/monkeypatch.rst

```diff
@@ -90,7 +266,7 @@ to do this using the ``setenv`` and ``delenv`` method. Our example code to test:
 
 .. code-block:: python
 
-    # contents of our original code file e.g. code.py
+    # Contents of our original code file e.g. code.py.
```

Comment:
I would leave all those "contents -> Contents" changes lone, just for consistency with the rest of the docs.

Reply:
Great catch, also ensured `monkeypatch` is lowercase everywhere except when it is the first word in the headers.

Reply:
I still see comments starting with an upper case letter... would you mind changing all comments to start with lowercase? This is the pattern we use through the docs and code. 👍

Reply:
> I still see comments starting with an upper case letter... would you mind changing all comments to start with lowercase? This is the pattern we use through the docs and code. 

That's new to me also - I see that there are comments without punctuation and starting lowercased, but when using punctuation / full sentences they should be capitalized (like above).

Reply:
full sentences yes, but in one-liners like this we don't use punctuation and start with a lowercase letter (at least that's what I see browsing the docs quickly now)

Reply:
Well, you will also see no typing hints currently - but maybe soon.. ;)
(what I want to say is that style can be improved, and I prefer it like above also)

Reply:
Well I won't get into that discussion right now then, the current style is fine.

@EvanKepner could you just squash and rebase your commits then? Thanks!

Reply:
No problem at all, squash/rebase coming soon. Also @nicoddemus sorry I misread your original comment and thought you wanted all comment lines capitalized but without punctuation unless part of a larger sentence :sweat_smile: 

I'm a fan of generally strict type hints and full doc-strings for each function. It's a difficult challenge with documentation because you don't want to add additional syntax that could confuse a reader. These tutorial examples are simplified to show how the patching works so I steered towards the least verbose form. 

Might be worth an update in the guidance to https://github.com/pytest-dev/pytest/blob/master/CONTRIBUTING.rst#write-documentation if you come to a consensus on standardized documentation style beyond the pre-commit checks for `black` in the docs. If you're seeking suggestions I like one sentence per line in RST with a hard-wrap (or just a hard-wrap at something sensible) to make suggested changes and PR reviews easier.